policy/modules/admin/usermanage.te

```diff
@@ -69,7 +69,7 @@ role useradd_roles types useradd_t;
 # Chfn local policy
 #
 
-allow chfn_t self:capability { chown dac_override fsetid setgid setuid sys_resource };
+allow chfn_t self:capability { chown dac_override fsetid setgid setuid sys_ptrace sys_resource };
```

Comment:
sys_ptrace doesn't seem correct here.  Why does chfn_t need to ptrace?

Reply:
Not sure and I can't reproduce it so I'll remove it.